.github/CHANGELOG.md

```diff
@@ -1,6 +1,37 @@
-## Release 0.2.0 (development release)
+## Release 0.1.1 (current release)
 
-## Release 0.1.0 (current release)
+### New features since last release
+
+* The Job class now has a `metadata` property which, by convention, returns
+  information about job failures.
+  [(#15)](https://github.com/XanaduAI/xanadu-cloud-client/pull/15)
+
+### Improvements
+
+* The CI workflows are now triggered when a branch is merged into `main`.
+  [(#15)](https://github.com/XanaduAI/xanadu-cloud-client/pull/15)
+
+### Bug Fixes
+
+* On Windows, the XCC configuration file is now stored at `...\Xanadu\xanadu-cloud\.env`.
```

Comment:
Do we need the `...`?

Reply:
Maybe? I just wanted to communicate that the configuration file is not stored in the relative directory
```
Xanadu\xanadu-cloud\.env
```